install.py

```diff
@@ -5,243 +5,287 @@
-                           f"{WORKING_DIR}libedit.deb")
-
-os.system(f"cd {WORKING_DIR} && ar -vx libncurses.deb && tar -xf data.tar.xz")
-os.system(f"cd {WORKING_DIR} && ar -vx libedit.deb && tar -xf data.tar.xz")
-
-# copy libs
-if install_global:
-    os.system(f"sudo cp {WORKING_DIR}lib/{arch}/libncurses.so.6.4 /lib/libncurses.so.6.4")
-    os.system(f"sudo cp {WORKING_DIR}usr/lib/{arch}/libform.so.6.4 /usr/lib/libform.so.6.4")
-    os.system(f"sudo cp {WORKING_DIR}usr/lib/{arch}/libpanel.so.6.4 /usr/lib/libpanel.so.6.4")
-    os.system(f"sudo cp {WORKING_DIR}usr/lib/{arch}/libedit.so.2.0.70 /usr/lib/libedit.so.2.0.70")
-
-    os.system("sudo ln -s /lib/libncurses.so.6.4 /lib/libncurses.so.6")
-    os.system("sudo ln -s /usr/lib/libform.so.6.4 /usr/lib/libform.so.6")
-    os.system("sudo ln -s /usr/lib/libpanel.so.6.4 /usr/lib/libpanel.so.6")
-    os.system("sudo ln -s /usr/lib/libedit.so.2.0.70 /usr/lib/libedit.so.2")
-else:
-    os.system(f"mkdir -p {mojo_lib_path}")
-
-    os.system(f"cp {WORKING_DIR}lib/{arch}/libncurses.so.6.4 {mojo_lib_path}/libncurses.so.6")
-    os.system(f"cp {WORKING_DIR}usr/lib/{arch}/libform.so.6.4 {mojo_lib_path}/libform.so.6")
-    os.system(f"cp {WORKING_DIR}usr/lib/{arch}/libpanel.so.6.4 {mojo_lib_path}/libpanel.so.6")
-    os.system(f"cp {WORKING_DIR}usr/lib/{arch}/libedit.so.2.0.70 {mojo_lib_path}/libedit.so.2")
-
-# install mojo
-mojo = shutil.which(f"PATH=$PATH:{home}/.modular/pkg/packages.modular.com_mojo/bin/ mojo") is not None
-if mojo:
-    print("Mojo is already installed... cleaning up")
-    os.system(f"PATH=$PATH:{home}/.modular/pkg/packages.modular.com_mojo/bin/ modular clean")
-
-os.system(f"LD_LIBRARY_PATH=$LD_LIBRARY_PATH:{mojo_lib_path} modular install mojo")
-
-# fix crashdb directory not found:
-os.makedirs(f"{home}/.modular/crashdb", exist_ok=True)
-
-
-def rc_path():
-    shell = param("SHELL")
-    if shell is not None:
-        match shell.split("/")[-1]:
-            case "bash":
-                return get_shell("bash", f"{param('HOME')}/.bashrc")
-            case "zsh":
-                return get_shell("zsh", f"{param('HOME')}/.zshrc")
-            case _:
-                path = input(
-                    "Please enter the path to your shell rc file (e.g. ~/.bashrc for bash) or press ENTER to skip:")
-                if path == "":
-                    return None
-                return path.replace("~", param("HOME"))
-
-    else:
-        return get_shell()
-
-
-def get_shell(found=None, file=None):
-    if found is not None:
-        yn = input(f"Found {found} shell, add exports to {file}? [y/N/other]")
-        yn = yn.lower()
-        if yn == "y":
-            return file
-        elif yn == "n":
-            return None
-        elif yn == "other" or yn == "o":
-            return get_shell()
-        elif yn == "":
-            print("Skipping...")
+
+        try:
+            os.makedirs(self.working_dir)
+        except OSError:
+            pass
```

Comment:
I'm not sure if it's good to pass all errors here, it probably should only let `FileExistsError`s pass

Reply:
Ok.

Reply:
Oh, I've actually changed it already. I'm going to push the current changes.